### ISSUE TITLE
node v10 install failed (outdated unix_dgram->nan)

### ISSUE THREAD

**Issue by igor-sky** [winstonjs/winston-syslog#99]
Because it uses outdated **unix_dgram** which depends on outdated **nan**

```
$ npm i winston-syslog@2.0.0

> unix-dgram@0.2.3 install /private/tmp/node_modules/winston-syslog/node_modules/unix-dgram
> node-gyp rebuild

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

  CXX(target) Release/obj.target/unix_dgram/src/unix_dgram.o
In file included from ../src/unix_dgram.cc:5:
In file included from ../../nan/nan.h:190:
../../nan/nan_maybe_43_inl.h:88:15: error: no member named 'ForceSet' in 'v8::Object'
  return obj->ForceSet(GetCurrentContext(), key, value, attribs);
```

**Comment by kafkahw** [comment]
I think this issue will be resolved by PR #105 if that PR gets merged

**Comment by mohd-akram** [comment]
#105 was merged. Any update on a new release? @santigimeno

**Comment by JBustin** [comment]
We couldn't upgrade node version cause of this bug, could you release it ?
Thx

**Comment by cyrus000** [comment]
will this fix be released soon?

**Comment by vinioliveira** [comment]
Any plans to release fix?

**Comment by croraf** [comment]
Is this fixed in 2.0.0, because I see 2.0.0 release is using latest unix-datagram which itself fixed the issue.
If so this issue should be closed.

**Comment by DABH** [comment]
Fixed in new v2.1.0 :balloon: !
https://github.com/winstonjs/winston-syslog
